README.md

```diff
@@ -154,7 +161,13 @@ module.exports = {
 
 ### `import`
 
-Type: `Boolean|Object`
+Type:
+
+```ts
+type import = boolean |
+ { filter:  (url:string, media:string, resourcePath:string) => boolean; }
```

Comment:
Looks like prettier doesn't work here, because we need space (i.e. ` `) after `:`

Reply:
It should be because `import` is a reserved keyword

Reply:
I've formatted it now (changed name and formatted and renamed to import)

Reply:
Let's use other name and add comment about it